src/Message/Attachment.php

```diff
@@ -16,15 +16,22 @@ class Attachment extends DataObject
      * @param string $title    The attachment title.
      * @param string $text     The attachment body text.
      * @param string $fallback A plain-text summary of the attachment.
+     * @param string $callback_id A unique text callback_id.
+     * @param string $color A color value
+     * @param string $pretext Pretext value
+     * @param array $fields Attachment fields
+     * @param array $actions Attachment actions
      */
-    public function __construct($title, $text, $fallback = null, $color = null, $pretext = null, array $fields = [])
+    public function __construct($title, $text, $fallback = null, $callback_id, $color = null, $pretext = null, array $fields = [], array $actions = [])
```

Comment:
If it is changed to `$callback_id = null` then it should also be moved to the end of the constructor, otherwise it will still be a breaking change.